fix unmotivated missing PReP partition warning (bsc#1081163)

wfeldt opened the pull request:
The code flow is just unfixably broken. This is the minimal fix addressing
the described problem.

shundhammer reviewed (approved):
Once Travis is happy, LGTM.

wfeldt committed:
fix unmotivated missing PReP partition warning (bsc#1081163)

The code flow is just unfixably broken. This is the minimal fix addressing
the described problem.

wfeldt committed:
release 3.2.19
